fieldservice/views/fsm_location.xml

```diff
@@ -22,7 +22,22 @@
         <field name="arch" type="xml">
             <form string="Location">
                 <sheet>
-                    <div class="oe_button_box" name="button_box"/>
+                    <div class="oe_button_box" position="inside">
+                        <button type="object"
+                                name="action_view_equipment"
+                                class="oe_stat_button"
+                                icon="fa-briefcase"
+                                attrs="{'invisible': [('equipment_count', '=', 0)]}" groups="base.group_user">
```

Comment:
Should be visible even if count is 0.

Groups should be set to the equipment group.